igor-core/src/main/java/com/netflix/spinnaker/igor/build/model/GenericArtifact.java

```diff
@@ -41,6 +41,14 @@ public GenericArtifact(String type, String name, String version, String referenc
     this.name = name;
     this.version = version;
     this.reference = reference;
+    this.fileName = String.format("%s:%s", name, version);
+  }
+
+  public GenericArtifact withUrl(String registry) {
+    String url = String.format("https://%s/v2/%s/blobs/%s", registry, name, reference);
```

Comment:
This class is a `GenericArtifact`, but this URL looks like it's GitHub-specific. Can you please explain how this will be used?

Reply:
good catch. fixed it.